### DISCUSSION TITLE
Version and external Server calls

### DISCUSSION THREAD

**Discussion by satyam-kr** [Tracardi/tracardi#411] [Q&A]
Will my exisiting tracardi setup will have any issue ? If new version comes up . And does internally tracardi calls any of tracardi servers?

**Comment by atompie** [comment]
Could you elaborate? An example you help us what you mean be external server call. Does it mean a call to an external API?

Tracardi is still in development, but we strive to keep it as compatible with older versions as possible. It all depends on the current version you are using. We have changed a bit the data schema so new version may require data reindex. 

Tracardi internally does not call other tracardi servers. Unless you want it to send some data with a remote API plugin, and you set it to call tracardi.
